index.js

```diff
@@ -111,6 +111,28 @@ Babel.prototype.optionsHash = function() {
       hash[key] = (typeof value === 'function') ? (value + '') : value;
     }
 
+    if (options.plugins) {
+      hash.plugins = [];
+
+      for (var i = 0; i < options.plugins.length; i++) {
+        var plugin = options.plugins[i];
+
+        // do not look for `._augmentCacheKey` on string plugins
+        var pluginType = typeof plugin;
+        var pluginAugmentsCacheKey = (
+          (pluginType === 'object' || pluginType === 'function')
+            && typeof plugin._augmentCacheKey === 'function'
+        );
+        if (pluginType === 'string') {
+          hash.plugins.push(plugin);
+        } else if (pluginAugmentsCacheKey) {
+          hash.plugins.push(plugin._augmentCacheKey());
+        } else {
+          hash.plugins.push(pluginType === 'function' ? plugin + '' : plugin);
```

Comment:
Ya, but we have to add *something*. I was simply emulating what was being done above for the main options keys.